tripzone/src/components/Card/BookNowCard/BookCard.jsx

```diff
@@ -0,0 +1,129 @@
+import "./BookCard.scss";
+import { AppConstants } from "../../../constants/App.constants";
+import Button from "../../Button/Button";
+import { useState, useEffect } from "react";
+import prime from "../../../assets/prime.png";
+import classNames from "classnames";
+import { useNavigate } from "react-router-dom";
+
+const { BOOK_NOW } = AppConstants.COMMONS;
+
+function BookCard({ ticketAmt }) {
+  console.log("CONTAINER - BOOK NOW CARD");
+  const [membershipDiscount, setMembershipDiscount] = useState(10);
+  const [taxAmount, setTaxAmount] = useState(5);
+  const [primeUser, setPrimeUser] = useState(false);
+  const ticketAmtNum = Number(ticketAmt.replace(/[^0-9.-]+/g, ""));
+  const navigate = useNavigate();
+
+  // Used to set if user is prime or not
+
+  useEffect(() => {
+    const storedUser = localStorage.getItem("user");
```

Comment:
To know whether user is primer or not ,  instead of local storage , could you come up react concept itself

Reply:
Local Storage is used considering page reload as context is being re-initialized and the user data is lost